build.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
```

Comment:
I suspect a great deal of this file can be cleaned up.  Since we are just building managed code, I would guess that it just passes everything to `build-managed.sh` (FWIW, I *do* like having both `build.sh` and `build-managed.sh`).